front/app/components/FormBuilder/utils.tsx

```diff
@@ -124,6 +124,22 @@ export function getAdditionalSettings(
           />
         </>
       );
+    case 'ranking':
+      return (
+        <>
+          <ConfigSelectWithLocaleSwitcher
+            name={`customFields.${field.index}.options`}
+            locales={locales}
+            platformLocale={platformLocale}
+            inputType={field.input_type}
+          />
+          <SelectSettings
+            inputType={field.input_type}
```

Comment:
I considered renaming these components to something more general (since Ranking isn't exactly "selection", but it kind of is as well? :thinking: ). I settled on just leaving them as-is, but let me know if you disagree :)

Reply:
Sounds good! I'll re-name them then :+1: